File: lib/JFFUI/UIView/UIView+AddSubviewAndScale.m
#import "UIView+AddSubviewAndScale.h"

@implementation UIView (AddSubviewAndScale)

-(void)addSubviewAndScale:( UIView* )view_
{
    [ view_ removeFromSuperview ];

    view_.frame = self.bounds;
    view_.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
    [ self addSubview: view_ ];
}

@end

File: iAsync.Swift/lib/JUI/JUI/UIView/UIView+AddSubviewAndScale.swift
//
//  UIView+AddSubviewAndScale.swift
//  JUI
//
//  Created by Vladimir Gorbenko on 03.01.15.
//  Copyright (c) 2015 EmbeddedSources. All rights reserved.
//

import UIKit

public extension UIView {

    func addSubviewAndScale(view: UIView) {
        
        view.removeFromSuperview()
    
        view.frame = self.bounds
        view.autoresizingMask = .FlexibleWidth | .FlexibleHeight
        addSubview(view)
    }
}
